tests/billing_groups_test.js

```diff
@@ -11,82 +11,72 @@ const resource_endpoint = "/billing_groups",
 
 const prism = new Prism(specFile, lobUri, process.env.LOB_API_TEST_TOKEN);
 const payload = {
-  name: "Open Api test",
-  description: "Open Api test",
+  name: "OpenAPI Test",
+  description: "OpenAPI testing for Billing Groups",
 };
-let create = null;
 
 // contract tests happy path
-test("create, list, read, replace, update, then delete a billing_group", async function (t) {
-  create = await prism
-    .setup()
+test.serial.before("create a billing_group", async function (t) {
```

Comment:
note that we gotta change this in the generator if we're committing to the "separate tests for create, list, and delete" structure

Reply:
Correct.